app/views/revocation/permissionWithdrawn.scala.html

```diff
@@ -1,16 +1,15 @@
-@(application: ApplicationDetails)(implicit request: Request[_])
+@()(implicit request: Request[_])
 
 @body = {
     <div class="content__body">
         <header>
             <h1>Authority removed</h1>
         </header>
 
-        <p data-withdrawn-message>
-            <strong>@{application.name}</strong> no longer has authority to interact with HMRC on your behalf.
-        </p>
+        <p data-withdrawn-message>This application no longer has authority to interact with HMRC on your behalf.</p>
+
+        <p>You can grant authority again in the application itself.</p>
```

Comment:
While making sure the new copy is fit for purpose, Ash and Alex agreed that this should be changed to normal paragraph (that was subsequently agreed with Adreea as well)